src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs

```diff
@@ -56,12 +56,16 @@ private static IConfigurationRoot BuildConfiguration()
                 .AddJsonFile("appsettings.json", optional: true)
                 .AddEnvironmentVariables();
 
-            return configurationBuilder.Build();
-        }
+            // replace the default environment source with our own
+            IConfigurationSource envVarsSource = configurationBuilder.Sources.OfType<EnvironmentVariablesConfigurationSource>().FirstOrDefault();
+            if (envVarsSource != null)
+            {
+                configurationBuilder.Sources.Remove(envVarsSource);
```

Comment:
Rather than remove this, why not just skip adding it altogether?

Reply:
How?

Reply:
Right above this -- remove `AddEnvironmentVariables()` from the `ConfigurationBuilder`.

Reply:
doh! done